tests/ui/pages/desktop/collections.py

```diff
@@ -0,0 +1,32 @@
+from pypom import Region
+from selenium.webdriver.common.by import By
+
+from base import Base
+
+
+class Collections(Base):
+    """ Collections page. """
+
+    _name_locator = (By.CSS_SELECTOR, ".item > .info > h3")
+
+    def wait_for_page_to_load(self):
+        self.wait.until(lambda _: self.collections)
```

Comment:
I don't think `self.collections[0].name` would ever be `None`. It's fine to wait on multiple conditions, which is what you're doing by using `and`. You could also have two independent waits. What we should avoid in page objects is hidden logic from the test, such as taking one or more different paths. An example would be a `login` method that first checked if we were logged in and returned early, which could mask an issue where a previous step failed to logout as expected.

Your previous logic was conditional waits based on the base URL, which is a bad idea for other reasons. The base URL is not a good indicator of if an element will or will not be present/displayed, and such things should work across deployments.